cmd/bancheck/config/config.go

```diff
@@ -0,0 +1,86 @@
+package config
+
+import (
+	"encoding/json"
+	"errors"
+	"io/ioutil"
+	"os"
+)
+
+// Config struct which contains an array of banned imports and function calls
+type Config struct {
+	Imports   []BannedImport   `json:"imports"`
+	Functions []BannedFunction `json:"functions"`
+}
+
+// BannedImport struct which contains a fully qualified import name,
+// message with additional information and a list of exemptions.
+type BannedImport struct {
+	Name       string      `json:"name"`
+	Msg        string      `json:"msg"`
+	Exemptions []Exemption `json:"exemptions"`
+}
+
+// BannedFunction struct which contains a fully qualified function name,
+// message with additional information and a list of exemptions.
+type BannedFunction struct {
+	Name       string      `json:"name"`
+	Msg        string      `json:"msg"`
+	Exemptions []Exemption `json:"exemptions"`
+}
+
+// Exemption struct which contains a justification and a path to allowed directory.
+type Exemption struct {
+	Justification string `json:"justification"`
+	AllowedDir    string `json:"allowedDir"`
+}
+
+// Read reads configs from all files and concatenates them into one object.
+func Read(files []string) (*Config, error) {
+	imports := make([]BannedImport, 0)
```

Comment:
Use zero values for slices: https://github.com/golang/go/wiki/CodeReviewComments#declaring-empty-slices.

Reply:
Done